apps/condo/domains/common/utils/mail.spec.js

```diff
@@ -0,0 +1,66 @@
+const { normalizeEmail }  = require('./mail')
+
+describe('normalizeMail()', () => {
+    test('no data', () => {
+        expect(normalizeEmail()).toBeUndefined()
+        expect(normalizeEmail(null)).toBeUndefined()
+    })
+
+    test('empty trimmed data', () => {
+        expect(normalizeEmail('')).toBeUndefined()
+        expect(normalizeEmail('      ')).toBeUndefined()
+        expect(normalizeEmail('\t\t\t   ')).toBeUndefined()
+    })
+
+    test('correct lowering data', () => {
+        expect(normalizeEmail('asd@asd.asd')).toEqual('asd@asd.asd')
+        expect(normalizeEmail('ASD@asd.asd')).toEqual('asd@asd.asd')
+        expect(normalizeEmail('asd@asD.aSd')).toEqual('asd@asd.asd')
+        expect(normalizeEmail('uBiIcAOnBoardINGoVVV2007@sber.doma')).toEqual('ubiicaonboardingovvv2007@sber.doma')
+    } )
+
+    test('multiple @  test', () => {
+        expect(normalizeEmail('asd@asd@ads')).toBeUndefined()
+        expect(normalizeEmail('asd@asd.asd')).toEqual('asd@asd.asd')
```

Comment:
It seems, like this line is redundant, because another test case have it.

Reply:
removed :)